code-challenge/src/components/Article/Article.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Article.scss';
+import {  Divider, ListItem, ListItemText } from '@material-ui/core';
+
+export interface ArticleListProps {
+    title: string;
+    byline: string;
+    section: string;
+}
+
+const Article = ({ title, byline, section }: ArticleListProps) => (
+  <div className="Article" data-testid="Article">
```

Comment:
```suggestion
  <div className="Article">
```

Reply:
So the library I used for component generation created this, and I believe the concept is (and something I've encountered when doing E2E automation) that you want your test selectors to be stable to avoid redo-ing selectors for those tests each time a class changes. I think if the tests actually worked in this project (I didn't leave myself enough time to flesh those out) I might suggest keeping this attribute

Reply:
Huh, the reasoning is sound but it still seems odd to commit what looks like dev/debug code. I wonder if it's configurable to use the className or other attribute.